### PR TITLE
Layer-Sagemaker integration doc example

### PR DESCRIPTION
**Pull request by pm3310**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
...
+
+Layer provides Machine Learning practitioners and organizations the ability to log, display, compare and share their Datasets, Models and Project Documentation in one single place.  It helps you create production-grade ML pipelines with a seamless local<>cloud transition while enabling collaboration with semantic versioning, extensive artifact logging and dynamic reporting.
+
+The steps below explain how to take advantage of both platforms, Layer and Sagemaker. At the end, you will have trained your model using the Sagemaker platform, and integrate with Layer to version your Dataset and register your model to Layer.
+
+### Step 1: Create a Layer Account
+
+Go to [Layer](https://layer.ai/) and click the signup button to create an account. There's a free tier option.
+
+### Step 2: Clone Machine Learning demo repository
```
Comment (mwitiderrick) [inline docs/index.md:935]: machine learning

File: docs/index.md [hunk 2]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
 Now you should be able to see data coming in on Aporia dashboards.
 
 
+## Layer
+
+Layer provides Machine Learning practitioners and organizations the ability to log, display, compare and share their Datasets, Models and Project Documentation in one single place.  It helps you create production-grade ML pipelines with a seamless local<>cloud transition while enabling collaboration with semantic versioning, extensive artifact logging and dynamic reporting.
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: done

File: docs/index.md [hunk 3]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
...
+
+The steps below explain how to take advantage of both platforms, Layer and Sagemaker. At the end, you will have trained your model using the Sagemaker platform, and integrate with Layer to version your Dataset and register your model to Layer.
+
+### Step 1: Create a Layer Account
+
+Go to [Layer](https://layer.ai/) and click the signup button to create an account. There's a free tier option.
+
+### Step 2: Clone Machine Learning demo repository
+
+You're going to clone and train a Machine Learning codebase to train a classifier for the Iris data set.
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: Done

File: docs/index.md [hunk 4]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
 Now you should be able to see data coming in on Aporia dashboards.
 
 
+## Layer
+
+Layer provides Machine Learning practitioners and organizations the ability to log, display, compare and share their Datasets, Models and Project Documentation in one single place.  It helps you create production-grade ML pipelines with a seamless local<>cloud transition while enabling collaboration with semantic versioning, extensive artifact logging and dynamic reporting.
+
+The steps below explain how to take advantage of both platforms, Layer and Sagemaker. At the end, you will have trained your model using the Sagemaker platform, and integrate with Layer to version your Dataset and register your model to Layer.
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: Done

File: docs/index.md [hunk 5]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
...
+        from sklearn.svm import SVC
+        
+        layer.login_with_api_key("Go to https://docs.app.layer.ai/docs/getting-started/log-in#log-in-with-an-api-key to get your API KEY")
+        layer.init("sagemaker_iris")
+
+    Don't forget to replace the API key following the instructions [here](https://docs.app.layer.ai/docs/getting-started/log-in#log-in-with-an-api-key)
+
+### Step 4: Build Docker image
+
+It's time to build the Docker image that will contain the Machine Learning codebase:
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: Done

File: docs/index.md [hunk 6]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
...
+
+    sagify push
+
+### Step 7: Create S3 bucket and Upload Training Data
+
+Make sure to create an S3 bucket with a name of your choice, for example: `layer-sagify-demo`, and execute `sagify cloud upload-data -i data/ -s s3://layer-sagify-demo/training-data` to upload training data to S3.
+
+### Step 8: Train on AWS SageMaker
+
+Execute `sagify cloud train -i s3://layer-sagify-demo/training-data/ -o s3://layer-sagify-demo/output/ -e ml.m4.xlarge` to train the Machine Learning model on SageMaker. This command will use the pushed Docker image.
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: done

File: docs/index.md [hunk 7]
```diff
@@ -922,6 +922,183 @@ Run `sagify local deploy` and then run the following curl command to call the in
...
+
+### Step 9: Layer Dashboard
+
+When training finishes, you can go to your `My Projects` at Layer, and find the sagemaker-iris project. There will be 2 created assets in this project. One asset for your Dataset, and another for your Model. The advantage here is that you have used the power of Sagemaker for leveraging training resources, and the power of Layer for reproducible Machine Learning pipelines.
```
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md:1099]: added

File: docs/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pm3310) [reply docs/index.md]: done

File: docs/index.md (hunk 1 above)
Comment (pm3310) [reply docs/index.md]: done

**Comment by pm3310** [comment]
Hey @mwitiderrick , apologies for the late reply. I was on holiday. Pls, take a final look and then I'll merge it to master branch

**Pull request closed without merging**